mergents-guide-to-the-galaxy.iml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file looks like a intellij settings file. Normally you shouldn't commit these as they are local to your environments. Normal practice is to put all intellij/eclipse settings file to add to `.gitignore` file so that git will not track them